src/util/sanitizers.js

```diff
@@ -0,0 +1,7 @@
+function sanitizeIntegers(integers, req){
```

Comment:
Rename this function so it is clear that only path parameters will be sanitized.